### PR TITLE
Add padding for ol tag

### PR DESCRIPTION
**Pull request by jetfuel**
add decimal style for li tag under ol tag

https://github.com/PaddlePaddle/PaddlePaddle.org/issues/357

### CODE REVIEW

File: portal/portal/static/sass/content.sass [hunk 1]
```diff
@@ -404,9 +404,14 @@
         margin-bottom: 0
 
     ol
+        padding: 24px
```
Comment (varunarora) [inline portal/portal/static/sass/content.sass]: So we use `em`. Perhaps you could use an em value? I think the suggestion is 2em from the adopted stylesheet.

File: portal/portal/static/sass/content.sass [hunk 2]
```diff
@@ -404,9 +404,14 @@
...
+
         ol
             list-style-type: lower-roman
 
+        li
```
Comment (varunarora) [inline portal/portal/static/sass/content.sass]: So just gotta be a little careful. This applies the styling to the blanket element `li` which is the child of any `ol`. Perhaps you wish to just the first-level children of `ol` by doing it this way:
```
> li
    list-style: decimal
```

Comment (jetfuel) [reply portal/portal/static/sass/content.sass]: Good suggestion. I want it to as strict as possible. This definitely helps that.

**Comment by jetfuel** [comment]
@varunarora updated.

**Pull request merged**